### DISCUSSION TITLE
Hide properties from Snoop?

### DISCUSSION THREAD

**Discussion by Khaos66** [snoopwpf/snoopwpf#304] [Q&A] [answered]
Is it possible to hide properties from snoop? Via Attributes or smth?
Or mark them for lazy loading?

**Comment by batzen** [comment]
Do you mean by filtering from a user perspective, or preventing Snoop from showing something from the application developer perspective?

**Reply by Khaos66** [reply]
Hi batzen, I mean from the developer side. I have some properties that use lazy loading to load some external data on first access. When I use snoop and select the control the lazy loading triggers. I don't want that ;)

**Reply by batzen** [reply]
Ah. Now i get it.
There is currently no way to tell Snoop about some being lazy, but it sounds like a nice addition.
The most obvious solution would be some new attribute like "IsLazy" or "SnoopLazy".
If Snoop encounters such attribute it will show the property, but instead of it's value will show a button which loads the value when clicked.
What do you think?

**Reply by Khaos66** [reply]
Sounds great! That would do.

**Reply by Khaos66** [reply]
@batzen is this something you can make happen anytime soon? Should I create an issue or would you like to specify this yourself?

**Comment by batzen** [comment, marked as answer]
Fixed by #314
